src/arviz_plots/plots/combine.py

```diff
@@ -0,0 +1,95 @@
+import re
+from importlib import import_module
+
+import xarray as xr
+from arviz_base import rcParams
+
+from arviz_plots import PlotCollection
+from arviz_plots.plots.utils import process_group_variables_coords
+
+def get_valid_arg(key, value, backend):
+    plot_backend = import_module(f"arviz_plots.backend.{backend}")
+    key_matcher = "color" if key in {"facecolor", "edgecolor"} else key
+    if isinstance(value, str) and re.match(key_matcher +"_[0-9]+", value):
+        _, index = value.rsplit("_", 1)
+        index = int(index)
+        return plot_backend.get_default_aes(key, index+1)[index]
+    return value
+
+
+def backendize_kwargs(kwargs, backend):
+    return {key: get_valid_arg(key, value, backend) for key, value in kwargs.items() if key != "function"}
+
+def render(da, target, backend, **kwargs):
+    plot_backend = import_module(f"arviz_plots.backend.{backend}")
+    plot_kwargs = da.item()
+    plot_fun_name = plot_kwargs["function"]
+    plot_kwargs = backendize_kwargs(plot_kwargs, backend)
+    return getattr(plot_backend, plot_fun_name)(target=target, **{**plot_kwargs, **kwargs})
```

Comment:
This probably needs a better place to live in. Also happy to get better naming suggestions.